services/crates/crates-msrv.service.js

```diff
@@ -0,0 +1,78 @@
+import { NotFound, pathParams } from '../index.js'
+import {
+  BaseCratesService,
+  description as cratesIoDescription,
+} from './crates-base.js'
+
+const description = `
+${cratesIoDescription}
+
+MSRV is a crate's minimum suppported rust version,
+the oldest version of Rust supported by the crate.
+See the [Cargo Book](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field)
+for more info.
+`
+
+export default class CratesMSRV extends BaseCratesService {
+  static category = 'platform-support'
+  static route = {
+    base: 'crates/msrv',
+    pattern: ':crate/:version?',
+  }
+
+  static openApi = {
+    '/crates/msrv/{crate}': {
+      get: {
+        summary: 'Crates.io MSRV',
+        description,
+        parameters: pathParams({
+          name: 'crate',
+          example: 'serde',
+        }),
+      },
+    },
+    '/crates/msrv/{crate}/{version}': {
+      get: {
+        summary: 'Crates.io MSRV (version)',
+        description,
+        parameters: pathParams(
+          {
+            name: 'crate',
+            example: 'serde',
+          },
+          {
+            name: 'version',
+            example: '1.0.194',
+          },
+        ),
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'MSRV', color: 'blue' }
```

Comment:
Label should be lowercase. See https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines

```suggestion
  static defaultBadgeData = { label: 'msrv', color: 'blue' }
```

Reply:
Should acronyms still be lowercase? MSRV is an acronym for "minimum supported rust version".
I couldn't find any badges with labels that are acronyms (but maybe I'm not looking hard enough), so I don't if there is precedent for this.

I personally like the uppercase more, but I have no issue with lowercasing it.

samples:
![MSRV](https://img.shields.io/badge/MSRV-1.70+-blue)
![msrv](https://img.shields.io/badge/msrv-1.70+-blue)

Reply:
Most of the examples I can think of are where the badge label is the name of a service, e.g:

Comprehensive Perl Archive Network: ![](https://img.shields.io/cpan/v/Config-Augeas)
Arch linux User Repository: ![](https://img.shields.io/aur/version/visual-studio-code-bin)
Node Package Manager: ![](https://img.shields.io/npm/v/got)

but label is always lowercase

Reply:
ok! thanks for the examples. i guess it can be overridden with `?label=` anyway :shrug: